src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -0,0 +1,175 @@
+package dotty.tools.dotc
+package printing
+
+import core._
+import Texts._, Types._, Flags._, Names._, Symbols._, NameOps._, Contexts._
+import collection.mutable
+import collection.Map
+import Decorators._
+import scala.annotation.switch
+import scala.util.control.NonFatal
+import reporting.Diagnostic
+
+object Formatting {
+
```

Comment:
I'm wondering if it wouldn't be nicer to move the string interpolators here, it's not like anyone is going to use the a formatter by hand without using the corresponding interpolator